src/main/java/net/minestom/server/entity/Entity.java

```diff
@@ -1565,7 +1566,7 @@ private Duration getSynchronizationCooldown() {
 
     @Override
     public <T> @Nullable T getTag(@NotNull Tag<T> tag) {
-        return tag.read(nbtCompound);
+        return tag.read(nbtCompound.toCompound());
```

Comment:
I assume that the conversion is unnecessary?

Reply:
Indeed, will fix in next commit